src/components/MovieCard/moviecard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './moviecard.css';
+
+const MovieCard = ({ title, description }) => (
+  <div className="MovieCard">
+    <img src="https://i.pinimg.com/originals/f3/a2/0d/f3a20d7df90d3b4a4167a419a0566ff3.jpg" alt="movieImage" />
```

Comment:
`alt="movieImage"`
that one will be shown if image has failed to load or no opportunity to show it. That must be human readable

Reply:
fixed